persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/StructuralVariantMyBatisRepository.java

```diff
@@ -0,0 +1,23 @@
+package org.cbioportal.persistence.mybatis;
+
+import org.cbioportal.model.StructuralVariant;
+import org.cbioportal.persistence.StructuralVariantRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class StructuralVariantMyBatisRepository implements StructuralVariantRepository {
+
+    @Autowired
+    private StructuralVariantMapper structuralVariantmapper;
```

Comment:
structuralVariantmapper pls changed to structuralVariantMapper. Pls maintain the camel case code styling.